[2025-04-13T00:30:03.943+0000] {processor.py:186} INFO - Started process (PID=10110) to work on /opt/airflow/dags/scraping_dag.py
[2025-04-13T00:30:03.943+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/scraping_dag.py for tasks to queue
[2025-04-13T00:30:03.945+0000] {logging_mixin.py:190} INFO - [2025-04-13T00:30:03.944+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/scraping_dag.py
[2025-04-13T00:30:04.042+0000] {logging_mixin.py:190} INFO - [2025-04-13T00:30:04.039+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/scraping_dag.py", line 15, in <module>
    from src import Scrape_script
ImportError: cannot import name 'Scrape_script' from 'src' (/opt/airflow/dags/src/__init__.py)
[2025-04-13T00:30:04.044+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/scraping_dag.py
[2025-04-13T00:30:04.060+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/scraping_dag.py took 0.122 seconds
[2025-04-13T00:30:34.323+0000] {processor.py:186} INFO - Started process (PID=10179) to work on /opt/airflow/dags/scraping_dag.py
[2025-04-13T00:30:34.324+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/scraping_dag.py for tasks to queue
[2025-04-13T00:30:34.325+0000] {logging_mixin.py:190} INFO - [2025-04-13T00:30:34.325+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/scraping_dag.py
[2025-04-13T00:30:34.421+0000] {logging_mixin.py:190} INFO - [2025-04-13T00:30:34.418+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/scraping_dag.py", line 15, in <module>
    from src import Scrape_script
ImportError: cannot import name 'Scrape_script' from 'src' (/opt/airflow/dags/src/__init__.py)
[2025-04-13T00:30:34.423+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/scraping_dag.py
[2025-04-13T00:30:34.441+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/scraping_dag.py took 0.122 seconds
[2025-04-13T00:31:04.494+0000] {processor.py:186} INFO - Started process (PID=10247) to work on /opt/airflow/dags/scraping_dag.py
[2025-04-13T00:31:04.495+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/scraping_dag.py for tasks to queue
[2025-04-13T00:31:04.496+0000] {logging_mixin.py:190} INFO - [2025-04-13T00:31:04.496+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/scraping_dag.py
[2025-04-13T00:31:04.594+0000] {logging_mixin.py:190} INFO - [2025-04-13T00:31:04.590+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/scraping_dag.py", line 15, in <module>
    from src import Scrape_script
ImportError: cannot import name 'Scrape_script' from 'src' (/opt/airflow/dags/src/__init__.py)
[2025-04-13T00:31:04.595+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/scraping_dag.py
[2025-04-13T00:31:04.614+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/scraping_dag.py took 0.125 seconds
[2025-04-13T00:31:34.848+0000] {processor.py:186} INFO - Started process (PID=10311) to work on /opt/airflow/dags/scraping_dag.py
[2025-04-13T00:31:34.849+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/scraping_dag.py for tasks to queue
[2025-04-13T00:31:34.851+0000] {logging_mixin.py:190} INFO - [2025-04-13T00:31:34.850+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/scraping_dag.py
[2025-04-13T00:31:34.963+0000] {logging_mixin.py:190} INFO - [2025-04-13T00:31:34.959+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/scraping_dag.py", line 15, in <module>
    from src import Scrape_script
ImportError: cannot import name 'Scrape_script' from 'src' (/opt/airflow/dags/src/__init__.py)
[2025-04-13T00:31:34.965+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/scraping_dag.py
[2025-04-13T00:31:34.985+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/scraping_dag.py took 0.143 seconds
[2025-04-13T00:32:05.805+0000] {processor.py:186} INFO - Started process (PID=10381) to work on /opt/airflow/dags/scraping_dag.py
[2025-04-13T00:32:05.806+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/scraping_dag.py for tasks to queue
[2025-04-13T00:32:05.807+0000] {logging_mixin.py:190} INFO - [2025-04-13T00:32:05.807+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/scraping_dag.py
[2025-04-13T00:32:05.905+0000] {logging_mixin.py:190} INFO - [2025-04-13T00:32:05.902+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/scraping_dag.py", line 15, in <module>
    from src import Scrape_script
ImportError: cannot import name 'Scrape_script' from 'src' (/opt/airflow/dags/src/__init__.py)
[2025-04-13T00:32:05.907+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/scraping_dag.py
[2025-04-13T00:32:05.926+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/scraping_dag.py took 0.127 seconds
